Documentation/gettingstarted/bgp-control-plane.rst

```diff
@@ -0,0 +1,324 @@
+Cilium BGP Control Plane
+========================
+
+Usage
+-----
+
+Currently a single flag in the ``Cilium Agent`` exists to turn on the
+``BGP Control Plane`` feature set.
+
+::
+
+   --enable-bgp-control-plane=true
+
+When set to ``true`` the ``BGP Control Plane`` ``Controllers`` will be
+instantiated and will begin listening for ``CiliumBGPPeeringPolicy``
+events.
+
+CiliumBGPPeeringPolicy CRD
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All ``BGP`` peering topology information is carried in a
+``CiliumBGPPeeringPolicy`` CRD.
+
+``CiliumBGPPeeringPolicy`` can be applied to one or more nodes based on
+its ``nodeSelector`` fields.
+
+A Cilium node may only have a single ``CiliumBGPPeeringPolicy`` apply to
+it and if more than one does, it will apply no policy at all.
+
+Each ``CiliumBGPPeeringPolicy`` defines one or more
+``CiliumBGPVirtualRouter`` configurations.
+
+When these CRDs are written or read from the cluster the ``Controllers``
+will take notice and perform the necessary actions to drive the
+``BGP Control Plane`` to the desired state described by the policy.
+
+The policy in ``yaml`` form is defined below:
+
+.. code-block:: yaml
+
+   apiVersion: "cilium.io/v2alpha1"
+   kind: CiliumBGPPeeringPolicy
+   metadata:
+    name: 01-bgp-peering-policy
+   spec: # CiliumBGPPeeringPolicySpec
+    nodeSelector:
+      matchLabels:
+        bgp-policy: a
+    virtualRouters: # []CiliumBGPVirtualRouter
+    - localASN: 64512
+      exportPodCIDR: true
+      neighbors: # []CiliumBGPNeighbor
+       - peerAddress: 'fc00:f853:ccd:e793::50/128'
+         peerASN: 64512
+
+Fields
+^^^^^^
+
+::
+
+   nodeSelector: Nodes which are selected by this label selector will apply the given policy
+    
+    virtualRouters: One or more peering configurations outlined below. Each peering configuration can be thought of as a BGP router instance.
+       
+       virtualRouters[*].localASN: The local ASN for this peering configuration
+       
+       virtualRouters[*].exportPodCIDR: Whether to export the private pod CIDR block to the listed neighbors
+       
+       virtualRouters[*].neighbors: A list of neighbors to peer with
+           neighbors[*].peerAddress: The address of the peer neighbor
+           neighbors[*].peerASN: The ASN of the peer   
+
+.. note:: Setting unique configuration details of a particular
+instantiated virtual router on a particular Cilium node is explained
+in\ `Virtual Router Attributes <#Virtual%20Router%20Attributes>`__
```

Comment:
To fix the build:

```suggestion
.. note::

    Setting unique configuration details of a particular
    instantiated virtual router on a particular Cilium node is explained
    in `Virtual Router Attributes <#Virtual%20Router%20Attributes>`__
```

`.. note::` is a directive name, the content should go on the next line (usually, it's even separated with a blank line).

Reply:
If you ever re-push: There's still a backslash in here that could be removed (`in\ 'Virtual ...`). Pandoc added it because initially you had `*` for italics, and it needed to escape the space somehow.

Reply:
If you ever need to repush: You still have a trailing `\` here on the last line. I think Pandoc added it because initially you had italics ending here, and it needed to escape the space after the `*` delimiter.

Reply:
If you ever need to repush: You still have a trailing `\` here on the last line. I think Pandoc added it because initially you had italics ending here, and it needed to escape the space after the `*` delimiter.

Reply:
cool, fixed.